### ISSUE TITLE
Can create Issues

### ISSUE THREAD

**Issue by torik42** [matze-dd/Toy-131#1]
By the way, I can create issues (and PRs) and fork this repo. You might want to archive it (that’s in the same settings, where you can also transfer it).

**Comment by matze-dd** [comment]
Yes, thanks for the reminder. I did this already for the predecessor of YaLafi and had it on my own TODO list.
